packages/tenderly-core/src/internal/cli/commands/LoginCommand.ts

```diff
@@ -53,14 +63,20 @@ const validator = async function (value: string) {
 
 async function canAuthenticate(accessToken: string): Promise<boolean> {
   try {
+    logger.debug("Determining if the user can be authenticated with provided access token:", accessToken);
+
+    logger.trace("Making a call to the url...");
     const response = await axios.get(`${TENDERLY_API_BASE_URL}/api/v1/user`, {
```

Comment:
Are we sure we want to expose this `TENDERLY_API_BASE_URL` in the logs? I guess it's fine, right?